cli/src/main/java/com/devonfw/tools/ide/tool/cobigen/Cobigen.java

```diff
@@ -0,0 +1,32 @@
+package com.devonfw.tools.ide.tool.cobigen;
+
+import java.util.Set;
+
+import com.devonfw.tools.ide.context.IdeContext;
+import com.devonfw.tools.ide.tool.LocalToolCommandlet;
+import com.devonfw.tools.ide.tool.ToolCommandlet;
+import com.devonfw.tools.ide.tool.java.Java;
+
+/**
+ * {@link ToolCommandlet} for cobigen CLI (cobigen).
+ */
+public class Cobigen extends LocalToolCommandlet {
+
+  /**
+   * The constructor.
+   *
+   * @param context the {@link IdeContext}.
+   */
+  public Cobigen(IdeContext context) {
+
+    super(context, "cobigen", Set.of(TAG_CLOUD, TAG_IAC));
+  }
+
+  @Override
+  public boolean install(boolean silent) {
+
+    getCommandlet(Java.class).install();
```

Comment:
@jan-vcapgemini is that sufficient? IMHO cobigen CLI also requires `mvn` so we could replace `Java` with `Mvn` here (as `Mvn` already depends on `Java`).

Reply:
As @MattesMrzik  already mentioned, I agree that we need to replace Java with Mvn here.

Reply:
Changed from Java.class to Mvn,class